src/solidity-frontend/AST/solidity_ast.h

```diff
@@ -0,0 +1,18 @@
+#include <util/message/default_message.h>
+
+// For json parsing
+using json = nlohmann::json;
+
+class solidity_ast
```

Comment:
This class and if fact this new header file seems to be unused and unfinished. I would suggest removing it. If you want to keep it, please
- remove the global `using json = [...]` to reduce namespace pollution
- consider not dumping via `msg.debug` but `msg.status`, this makes debugging easier
- finish it (e.g. by adding a `to_string()` method) so it compiles when included somewhere

Reply:
This has been committed by mistake, I will remove